README.md

```diff
@@ -1,219 +1,121 @@
-
-[![Go Report Card](https://goreportcard.com/badge/github.com/keycloak/keycloak-operator)](https://goreportcard.com/report/github.com/keycloak/keycloak-operator)
-[![Coverage Status](https://coveralls.io/repos/github/keycloak/keycloak-operator/badge.svg?branch=main)](https://coveralls.io/github/keycloak/keycloak-operator?branch=main)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-
-# Legacy Keycloak Operator
-A Kubernetes Operator based on the Operator SDK for creating and syncing resources in Keycloak [WildFly distribution].
-
-**NOTE: This Operator is deprecated and will reach EOL with [Keycloak 20](https://www.keycloak.org/2022/03/releases).
-Consider migrating to the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator).**
+# Keycloak Realm Operator
+A Kubernetes Operator based on the Operator SDK for managing Realm and its sub-resources in Keycloak. This Operator is
+forked from the [legacy Keycloak Operator](https://github.com/keycloak/keycloak-operator) and stripped off of any functionality
+related to Keycloak Deployment as it was designed for the WildFly distribution of Keycloak and is not compatible with the
+new Quarkus distribution. The Realm Operator is meant as a **temporary workaround** to compensate for the missing
```

Comment:
@vmuzikar It'd be good to mention sth like that even in the description for these operators in Quay/DockerHub/OperatorHub, right? Probably could be done in a follow-up task, but IMHO it's quite important to have at least some description there. 

The thing is, I'm kinda afraid that could be a little bit confusing for the users as they see the `keycloak-operator` and `keycloak-realm-operator` in a hub together.

Reply:
Also, maybe add some guide to the keycloak-web??

Reply:
Sure, we'll have to prepare release notes for it. However, at this moment we don't plan to publish `keycloak-realm-operator` in OperatorHub as it is only a temporary workaround.